wasm_filter/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "filter"
+version = "0.1.0"
+authors = ["Eloy Coto <eloy.coto@gmail.com>"]
+edition = "2018"
+
+[dependencies]
```

Comment:
We can use `threescalers` here as an exercise in authorizing with 3scale.

Reply:
in a followup PR.